tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -424,6 +476,24 @@ where
             write!(writer, "{} ", fmt_level)?;
         }
 
+        if self.display_thread_name {
+            let current_thread = std::thread::current();
+            match current_thread.name() {
+                Some(name) => {
+                    write!(writer, "{} ", FmtThreadName::new(name))?;
+                }
+                // fall-back to thread id when name is absent and ids are not enabled
+                None if !self.display_thread_id => {
+                    write!(writer, "{:>2?} ", current_thread.id())?;
+                }
+                _ => {}
+            }
+        }
+
+        if self.display_thread_id {
+            write!(writer, "{:>2?} ", std::thread::current().id())?;
```

Comment:
Looking at the example output, I wonder if we want to pad thread IDs with 0s instead of a space, since they are numbers? I think
```
ThreadId(01)
```
might look nicer than
```
ThreadId( 1)
```
what do you think?

Reply:
Yep, Makes sense. Updated!